src/extensions/default/bramble/lib/MouseManagerRemote.js

```diff
@@ -1,6 +1,21 @@
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
 /* global addEventListener, removeEventListener, sessionStorage */
-(function(transport) {
```

Comment:
Don't move this line.  You need to wrap the entire module in a function like this.